src/coreclr/inc/sospriv.idl

```diff
@@ -465,12 +490,16 @@ interface ISOSDacInterface12 : IUnknown
 [
     object,
     local,
-    uuid(3176a8ed-597b-4f54-a71f-83695c6a8c5d)
+    uuid(3176a8ed-597b-4f54-a71f-83695c6a8c5e)
```

Comment:
I assume that ISOSDacInterface13 with the old uuid is new for .NET 8 so it is ok to change it here.

Reply:
Correct.

ISOSDacInterface13 is new in .Net 8, which is why I'm making an in-place change instead of adding 14.  Updating the uuid ensures that anyone who took a dependency on the previous sosdac13 won't get bitten by these changes and can distinguish the two.  For example, ClrMD currently carries both interfaces (https://github.com/microsoft/clrmd/pull/1133).  I plan to only support the RTM version after we ship .Net 8, but for now we support both in case it made it into a preview build.